packages/flutter_tools/test/general.shard/commands/build_android_test.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Maybe move these tests to `gradle_test.dart` since I don't think there's a `build_android.dart` file.  Ideally, we want to name test files such that they map to the file they are testing.

Reply:
These tests are more of integration tests, so it's hard to bind to a single file. The main focus is to test `android_builder.dart`, so I moved to `android_builder_test.dart`.